### DISCUSSION TITLE
Add UWP Games (Game Pass) to Sunshine

### DISCUSSION THREAD

**Discussion by ErikApption** [LizardByte/community#284] [Sunshine]
Would be nice to have native support for UWP games and add them without going through UWP Hook + Steam

**Comment by jmagar** [comment]
What I currently do is use steam rom manager to have the games added to steam, then in steam add a shortcut to desktop from the game you just added. Then in sunshine/Geforce experience use that shortcut to launch the game.

This also has the added benefit of being able to have separate controller configs for all your games without having to switch them.

**Reply by ErikApption** [reply]
steam rom manager - oh that is a nice one thanks for the mention. I've searched for many similar apps don't know why they have to have such obscure names. Anyway while Steam handles the controller mapping perfectly, it has some weird issues when in full screen mode. There is a known old bug that some games launch in the background when you have Steam big picture. Also, I don't think it is a stretch to have a tool like Rom Manager or BoilR (https://github.com/PhilipK/BoilR) add artwork + shortcut to Sunshine.

**Reply by rfh1987** [reply]
I just got started with Sunshine, and found my favorite method for adding Microsoft Store Games to Sunshine is through the "Sunshine App Export" add-on for Playnite.

**Reply by arobo81** [reply]
> I just got started with Sunshine, and found my favorite method for adding Microsoft Store Games to Sunshine is through the "Sunshine App Export" add-on for Playnite.

Could you possibly explain in a bit more detail how to go about that? I cant seem to be able to find the Sunshine App Export add-on either in playnite or the web.

**Reply by rfh1987** [reply]
@arobo81 
Here is a step-by-step...

1. Go to Playnite's website of add-ons: https://playnite.link/addons.html.
2. Search for "sunshine", and you'll see the "Sunshine app export" under the "Generic" category.
3. Download and install it to Playnite.
4. In Playnite, select the games you want exported (you can shift-click or ctrl-click for multiple selections).
5. Go to the Playnite menu (upper-left, controller icon), and click Extensions -> Sunshine App Export -> Export selected games.
6. Confirm where your Sunshine config file is (it defaults to the default Sunshine location).
7. Click "Export Games".
8. It will now be in your Sunshine list.

So, it isn't a sync.  It literally just manually adds games in Playnite to your Sunshine list.  I like to confirm it is in there by going to my Sunshine server's config page and viewing the "Applications" list, which is also where you can remove entries you no longer want appearing in Sunshine.  I've noticed that it usually takes a minute for it to show as available to stream on client devices, but this seems to be normal behavior.
